From 336643e8445168a90720bec0e0f04b5fce80ac85 Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Sun, 31 Aug 2014 18:50:23 +0200
Subject: [PATCH 4/4] missing: add IFF_MULTI_QUEUE

This was added in 3.8, but we should building with 3.7 headers.

Reported by Samuli Suominen <ssuominen@gentoo.org>.

Signed-off-by: Bert van Hall <bert.vanhall@avionic-design.de>
---
 src/shared/missing.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/shared/missing.h b/src/shared/missing.h
index 4ca8585..b3d0d31 100644
--- a/src/shared/missing.h
+++ b/src/shared/missing.h
@@ -555,6 +555,10 @@ static inline int setns(int fd, int nstype) {
 #define IPV6_UNICAST_IF 76
 #endif
 
+#ifndef IFF_MULTI_QUEUE
+#define IFF_MULTI_QUEUE 0x100
+#endif
+
 #ifndef IFF_LOWER_UP
 #define IFF_LOWER_UP 0x10000
 #endif
-- 
2.7.4

